src/plugins/network/index.js

```diff
@@ -192,6 +232,29 @@ export default class extends FlipperPlugin<State, *, PersistedState> {
   onRowHighlighted = (selectedIds: Array<RequestId>) =>
     this.setState({selectedIds});
 
+  copyRequestCurlCommand = () => {
+    const {requests} = this.props.persistedState;
+    const {selectedIds} = this.state;
+    // Ensure there is only one row highlighted.
+    if (selectedIds.length !== 1) {
+      return;
+    }
+
+    const request = requests[selectedIds[0]];
+    let command: string = `curl -v -X ${request.method}`;
+    command += ` "${request.url}"`;
```

Comment:
Do we need to apply some extra escaping here (and in the other places we insert user data)? It seems like it would be feasible to create a malicious cURL command by adding special shell characters here, e.g. `; rm -rf` or `$(cat /etc/passwd | curl ...`.

Reply:
Good call! [The Chrome inspector](https://github.com/ChromeDevTools/devtools-frontend/blob/a86c14edda076df6d273e97b61359e8e7fe8ff86/front_end/network/NetworkLogView.js#L1646-L1761) seems to do something. I'll try my hand at this